index.html

```diff
@@ -5302,18 +5243,16 @@ <h2 id="canmakepayment-protections">
           <code>canMakePayment()</code> protections
         </h2>
         <p>
-          The {{PaymentRequest/canMakePayment()}} and
-          {{PaymentRequest/hasEnrolledInstrument()}} methods have the potential
-          to expose user information that could be abused for fingerprinting
+          The {{PaymentRequest/canMakePayment()}} method has the potential to
+          expose user information that could be abused for fingerprinting
```

Comment:
I wonder if we can soften or remove this section. The new `canMakePayment()` definition returns much less information so realistically may not be a real fingerprinting threat today (though it can be in a future with a large number of payment apps).

```suggestion
          The {{PaymentRequest/canMakePayment()}} method provides feature detection for different payment.
          It may become a fingerprinting vector if in the future, a large number of payment methods are available.
```